app/assets/javascripts/simulator.js

```diff
@@ -0,0 +1,20 @@
+// This is a manifest file that'll be compiled into simulator.js, which will include all the files
+// listed below.
+//
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, or any plugin's
+// vendor/assets/javascripts directory can be referenced here using a relative path.
+//
+// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
+// compiled file. JavaScript code in this file should be added after the last require_* statement.
+//
+// Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
+// about supported directives.
+//
+//= require rails-ujs
+//= require_tree .
+//= require jquery
+//= require jquery_ujs
+//= require bootstrap-typeahead-rails
+//= require bootstrap/dist/js/bootstrap.min.js
+//= require popper.js/dist/popper.min.js
+//= require serviceworker-companion
```

Comment:
Do we need all of these files for simulator? Remove imports which are not required